app/src/main/java/protect/card_locker/importexport/Importer.kt

```diff
@@ -0,0 +1,36 @@
+package protect.card_locker.importexport
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import org.json.JSONException
+import protect.card_locker.FormatException
+import java.io.File
+import java.io.IOException
+import java.text.ParseException
+
+/**
+ * Interface for a class which can import the contents of a stream
+ * into the database.
+ */
+interface Importer {
+    /**
+     * Import data from the input stream in a given format into
+     * the database.
+     *
+     * @throws IOException
+     * @throws FormatException
```

Comment:
Not your fault, but this comment seems outdated.


```suggestion
     * @throws IOException
     * @throws FormatException
     * @throws InterruptedException
     * @throws JSONException
     * @throws ParseException
```